src/classes/Passage.ts

```diff
@@ -77,6 +83,7 @@ export class Passage {
     }
 
     /**
+     * @deprecated Getting your API key will be removed in the next major release.
```

Comment:
just since these are the only uses of second person in our documentation

```suggestion
     * @deprecated Getting the API key will be removed in the next major release.
```